samples/textDetection.js

```diff
@@ -15,19 +15,18 @@
 
 'use strict';
 
-const async = require('async');
 const fs = require('fs');
 const path = require('path');
-
+const {promisify} = require('util');
```

Comment:
after proposed refactoring `promisify`  is not needed any more, please remove it.